include/secp256k1_musig.h

```diff
@@ -7,348 +7,152 @@
 extern "C" {
 #endif
 
-#include <stdint.h>
+#include <stddef.h>
 
-/** This module implements a Schnorr-based multi-signature scheme called MuSig
- * (https://eprint.iacr.org/2018/068.pdf). It is compatible with bip-schnorr.
+/** This module implements a Schnorr-based multi-signature scheme called MuSig2
+ * (https://eprint.iacr.org/2020/1261, see Appendix B for the exact variant).
+ * Signatures are compatible with BIP-340 ("Schnorr").
  * There's an example C source file in the module's directory
  * (src/modules/musig/example.c) that demonstrates how it can be used.
  *
- * The documentation in this include file is for reference and may not be sufficient
- * for users to begin using the library. A full description of API usage can be found
- * in src/modules/musig/musig.md
+ * The module also supports BIP-341 ("Taproot") public key tweaking and adaptor
+ * signatures as described in
+ * https://github.com/ElementsProject/scriptless-scripts/pull/24
```

Comment:
Is it worth adding https://bitcoinops.org/en/topics/adaptor-signatures/ as a reference, since the information there is more organized / comprehensive than the PR discussion?

Reply:
Added the reference but note that the only description and proof sketch of that scheme is in the PR.